### PR TITLE
feat: download openmldb spark distribution in java cicd

### PR DESCRIPTION
**Pull request by tobegit3hub**
* Add script to download openmldb spark distribution
* Download and extract spark files in cicd of java

This is useful for unit tests of TaskManager which requires TaskManager with Spark to run test cases.

### CODE REVIEW

**Comment by tobegit3hub** [comment]
The time of CICD may increase about 8 minutes to download Spark file. Compare with previous result.

image

**Review by vagetablechicken — APPROVED** [review]

File: .github/workflows/cicd.yaml [hunk 1]
```diff
@@ -166,6 +167,7 @@ jobs:
       - name: start services
         run: |
           sh steps/ut_zookeeper.sh start
+          sh steps/download_openmldb_spark.sh $SPARK_HOME
```
Comment (aceforeverd) [inline .github/workflows/cicd.yaml:170]: pls update coverage.yml as well

Comment (tobegit3hub) [reply .github/workflows/cicd.yaml:170]: Okay.

**Comment by vagetablechicken** [comment]
image
cicd error

File: .github/workflows/cicd.yaml (hunk 1 above)
Comment (aceforeverd) [inline .github/workflows/cicd.yaml:170]: should be ${{ env.SPARK_HOME }}
in a workflow runner, the shell don't know the $SPARK_HOME variable

Comment (tobegit3hub) [reply .github/workflows/cicd.yaml:170]: It is the error of `coverage` which lacks of `SPARK_HOME`. This `$SPARK_HOME` works for `cicd.yaml` workflow.

Comment (aceforeverd) [reply .github/workflows/cicd.yaml:170]: I' wrong

**Comment by aceforeverd** [comment]
well, let's wait ⌚

**Review by vagetablechicken — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**